acid/features/status/controller.py

```diff
@@ -10,11 +10,12 @@
 @status.route('/status')
 @status.route('/status/<string:pipename>')
 def show_status(pipename=None):
-    zuul_url = current_app.config["zuul"]["url"]
-    zuul_endpoint = current_app.config['zuul']['status_endpoint']
+    config = current_app.config
+    zuul_url = config["zuul"]["url"]
```

Comment:
Single quotes are used to address a field in a dictionary almost everywhere in code.
Change double quotes to single for consistency.